src/app/workflow/workflow.component.html

```diff
@@ -96,6 +96,15 @@ <h3>
           >
             View Public
           </a>
+          <button 
+            mat-flat-button
+            id="DOIButton"
+            class="mr-2"
+            color="accent"
+           (click)="requestDOIForWorkflowVersion()"
+           [disabled]="(isRefreshing$ | async) || selectedVersion.doiURL"
+           [matTooltip]="createdoimessage">Create DOI
```

Comment:
Depending on how this looks on screen, if there's room, I wonder if it shouldn't say `Create Zenodo DOI`.

Reply:
![Screen Shot 2019-07-18 at 10 36 49 AM](https://user-images.githubusercontent.com/3650921/61479135-02ba1600-a948-11e9-843e-2b2b4aca0657.png)

Reply:
Ok, there's not enough room. :)